src/comparator/wrappers.py

```diff
@@ -203,7 +203,7 @@ def required(self):
         return WithLocation(
             required,
             self.source_code_locations,
-            self.path + (8, 1052),
+            self.path + (8, 1052, 0),
```

Comment:
what is the meaning of this `0`?

Reply:
The field_behavior annotation of a field is an array, and we look at the first element (which is 0) to get the location of the `field_behavior.required`.

Reply:
I don't know the full context of this, but the `field_behavior` option is repeated, meaning that the `required` item (if present at all) could be at any valid index. The behaviour should be to check every `field_behavior` item for the field, and it's required if any item equals `FieldBehavior.REQUIRED`.
However, I may have misunderstood what this code is doing...